doc/services/pm/device.rst

```diff
@@ -10,37 +10,73 @@ setting :kconfig:option:`CONFIG_PM_DEVICE` to ``y``. When this option is
-be able to make decisions related to managing device power.
-
-When using this type of device power management, the kernel can change CPU
-power states quickly when :c:func:`pm_system_suspend()` gets called. This is
-because it does not need to spend time doing device power management if the
-devices are already put in the appropriate power state by the application or
-component managing the devices.
+Device runtime power management is a collaborative effort involving
+device drivers, subsystems, and applications. While device drivers
+play a crucial role in directly controlling the power state of
+devices, the decision to suspend or resume a device can also be
+influenced by higher layers of the software stack.
+
+Each layer—device drivers, subsystems, and applications—can operate
+independently without needing to know about the specifics of the other
+layers. This is achieved because the subsystem uses reference count
+to check when it needs to suspend or resume a device.
+
+ - **Device drivers** are responsible for managing the
+   power state of devices. They interact directly with the hardware to
+   put devices into low-power states (suspend) when they are not in
+   use and bring them back (resume) when needed. Drivers should use the
+   device runtime power management APIs provided by Zephyr to control the
+   power state of devices.
+
+ - **Subsystems**, such as sensors, file systems,
+   and network, can also influence device power management.
+   Subsystems may have better knowledge about the overall system
+   state and workload, allowing them to make informed decisions about
+   when to suspend or resume devices. For example, a networking
+   subsystem may decide to keep a network interface powered on if it
+   expects network activity in the near future.
+
+ - **Applications** running on Zephyr can impact device
+   power management as well. An application may have specific
+   requirements regarding device usage and power consumption. For
+   example, an application that streams data over a network may need
+   to keep the network interface powered on continuously.
+
+Collaboration between device drivers, subsystems, and applications is
+key to efficient device power management. For example, a device driver
+may not know that a subsystem will perform a series of sequential
+operations that require a device to remain powered on. In such cases,
+the subsystem can use device runtime power management to ensure that
+the device remains in an active state until the operations are
+complete.
+
+By working together, device drivers, subsystems, and applications in
+Zephyr OS can optimize device power management to reduce overall
+system power consumption while meeting the requirements of the system
+and applications.
+
+Using this method, the CPU is able to change power states quickly when
+:c:func:`pm_system_suspend()` is called because it does not need to spend time
+suspending and resuming devices that are runtime enabled.
```

Comment:
can you provide a bit more explanations here? it's not clear to me what's the causality between the statement and what the rest of the section covered.
Also, I would start the sentence as "When using Device Runtime Power Management ..."